Example/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - Expecta (1.0.5)
-  - PINFuture (1.0.0)
+  - PINFuture (4.0.0)
```

Comment:
running `pod install` in the Example directory generated this change. Lmk if it's incorrect.